public/app/features/trails/DataTrailCard.tsx

```diff
@@ -110,5 +110,18 @@ function getStyles(theme: GrafanaTheme2) {
     actions: css({
       marginRight: theme.spacing(1),
     }),
+    wordwrap: css({
+      overflow: 'hidden',
+      overflowWrap: 'anywhere',
+    }),
   };
 }
+
+function truncateLongLabels(filter: { key: string; value: string }, maxlength?: number, suffix?: string) {
+  suffix = suffix ?? '...';
+  maxlength = maxlength ?? 80;
+
+  const label = `${filter.key}: ${filter.value}`;
+
+  return label.slice(0, maxlength) + suffix;
+}
```

Comment:
Instead of this function if you'd add following into the `description` CSS class, it will do the same job.
Could you please try it out?

```
overflow: 'hidden',
textOverflow: 'ellipsis',
whiteSpace: 'nowrap',
```

Reply:
Looks good!